indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -431,9 +438,23 @@ class BookingDetails extends React.Component {
             rows={2}
             required
           />
+          {isAccepted && occurrenceCount > 1 && (
+            <Checkbox
+              label={Translate.string(
+                'I understand that this will reject all occurrences of the booking.'
+              )}
+              onChange={(__, {checked}) => {
+                this.setState({
+                  acceptRejection: checked,
+                });
+              }}
+            />
+          )}
           <Button
             type="submit"
-            disabled={submitting || pristine || hasValidationErrors || submitSucceeded}
+            disabled={
+              submitting || pristine || hasValidationErrors || submitSucceeded || !acceptRejection
```

Comment:
Since we're inside a FinalForm there's probably a nicer way to actually mark the form as invalid until the checkbox has been checked, but I have to try this out myself first...

Reply:
BTW, I think this will currently break rejecting a pending booking, since this condition still runs, but there's no checkbox to select.